src/containers/PatientProfilePage.tsx

```diff
@@ -228,12 +229,15 @@ const PatientProfilePage = ({
             }}
           />
           <RadioSelector
-            labels={['Male', 'Female']}
+            labels={['M', 'F']}
```

Comment:
Could you use the enum here?
```suggestion
            labels={Object.keys(Gender)}
```